aries_vcx/src/core/profile/profile.rs

```diff
@@ -6,6 +6,7 @@ use aries_vcx_core::{
     anoncreds::base_anoncreds::BaseAnonCreds,
     ledger::base_ledger::{AnoncredsLedgerRead, AnoncredsLedgerWrite, IndyLedgerRead, IndyLedgerWrite},
     wallet::base_wallet::BaseWallet,
+    WalletHandle,
```

Comment:
hmmm, not sure about `Profile` re-exporting a type from indy/vdrtools. It implies that consumers of a Profile now have to import the whole vdrtools crate. 

The CI/CD "check variants" step should stop this. I wonder if you could just have `wallet_handle(..)` return `i32?` so that it's not exposing vdrtools

Reply:
Well, it did 😅 . I don't really like having to export this either, but:
1) This whole profile thing will go away in the near future.
2) It's a necessary evil for testing because the handle is needed for migration, no two ways about it.

I will accommodate it differently, though.

Reply:
I put the whole method behind the `migration` feature flag, as that's really the only place it is needed (for testing).

Reply:
Also, while we could have the method export the inner type, since the scope is limited to testing I think it's better to just have it behind the feature flag. I just really forgot it was there as I did that a couple of weeks ago 😅.

Reply:
kk sounds good.

good news, the check step was failing i believe; https://github.com/hyperledger/aries-vcx/actions/runs/5344789783/jobs/9689606086?pr=867#step:5:1246.

IMO it's v important that we continue to allow consumers to import aries_vcx without any vdrtools deps (or without any credx/modular-libs deps if they prefer, etc)

Reply:
Yeah, it did. God bless CI 😅 .
I agree with you regarding the deps. It was really just a snafu on my part as it was a quick addition when I started this since the wallet handle of the current `Indy` wallet was needed.